test/tests/ver-1g/ver-1g.i

```diff
@@ -0,0 +1,128 @@
+## Run this file with one of three input files to simulate various verification cases
+## 1. equal_conc.i        -> TMAP4 and TMAP7 equal concentration case
+## 2. diff_conc_TMAP4.i  -> TMAP4 different concentration case
+## 3. diff_conc_TMAP7.i  -> TMAP7 different concentration case
+## Example: ~/projects/TMAP8/tmap8-opt -i ver-1g.i equal_conc.i
```

Comment:
```suggestion
## Run this file with one of three input files to simulate various verification cases
## a. equal_conc.i        -> TMAP4 and TMAP7 equal concentration case
## b. diff_conc_TMAP4.i  -> TMAP4 different concentration case
## c. diff_conc_TMAP7.i  -> TMAP7 different concentration case
## Example: ~/projects/TMAP8/tmap8-opt -i ver-1g.i equal_conc.i
```
Thank you for adding this!! Great idea! I just edit it so that the cases have the same notation as in the documentation.